CHANGELOG.md

```diff
@@ -1,10 +1,40 @@
+## 0.9.0-alpha+10
+
+> NOTE: This code is considered production quality, but depends on angular:
+> 5.0.0-alpha+9. The alpha tag represents the evolving nature of the AngularDart
+> API, not code quality (5.0.0-alpha+9 is used in production Google apps).
+
+ * Add Material Slider component.
+ * Remove `visibility.all` from `@Component` annotations. This is potentially
+   BREAKING.
```

Comment:
This same item was listed for 0.9.0-alpha+9. Maybe this is meant to say:

> - Remove <ins>some</ins> ...

I see that some remain (e.g. in `lib/material_slider/material_slider.dart`).

Reply:
Done.